spec/takakuda_spec.rb

```diff
@@ -18,4 +18,20 @@
 
     # OK, all is Takakuda!
   end
+
+  describe '#takk?' do
+    it 'Integer is false' do
+      expect(1.takk?).to be_falsey
+    end
+    it 'Any String is false' do
+      expect('hoge'.takk?).to be_falsey
+    end
+
+    it '"Takakuda" is false' do
```

Comment:
This case will expect returns `true`, will not?
Please confirm it!

Reply:
@onunu
Thank you for review. I missed this sentence.
I fixed it!